app/src/__tests__/components/settings/SettingsPage.spec.tsx

```diff
@@ -20,7 +21,13 @@ describe('SettingsPage', () => {
   });
 
   const render = () => {
-    return renderWithProviders(<SettingsPage />, store);
+    return renderWithProviders(
+      <Routes>
```

Comment:
Had to wrap `SettingsPage` in a `Routes` component to define the parent `/settings/` path.